lib/spacekit-dynamic-dns.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+const Https = require('https');
+
+const CreateLogger = require('./create-logger');
+
+const log = CreateLogger('SpaceKitDynamicDns');
+
+/**
+ * SpaceKitDynamicDns provides a basic Dynamic DNS client. It connects to
+ * the SpaceKit service, pointing the hostname to your public IP address.
+ * It automatically updates the DNS entry when your IP address changes.
+ */
+class SpaceKitDynamicDns {
```

Comment:
Is having `spacekit` in the file name and class redundant. Would renaming the file to `lib/dynamic-dns.js` and the class name to `DynamicDns` be good?